spec/_plugins/alias_generator_spec.rb

```diff
@@ -28,7 +28,35 @@
       })
     
     author_index = Jekyll::AuthorIndex.new(site, "base", "author_dir", "Author") 
-    expect(author_index.data).to eq({ 'author' => "Author", 'description' => 'author: Author', "publications" => [video, post], 'title' => "Author"})
+    
+    expect(author_index.data['author']).to eq("Author")
+    expect(author_index.data['description']).to eq("author: Author")
+    expect(author_index.data['publications']).to include(video, post)
+    expect(author_index.data['publications']).not_to include(newsletter)
+    expect(author_index.data['title']).to eq("Author")
```

Comment:
I think the previous format of assertion (ie, comparing the entire object in one go rather than properties of the object in multiple lines) was better as it expresses the same amount and more in one line.

Reply:
This is debatable. For several reasons I prefer the modified version
1. Avoid the concern of sorting in this test case
2. Easy to read, especially the include and not include.
3. No implicit assumption of the eq operation's implementation. May not be a thing for Ruby, but quite a concern for Java.

Reply:
That's fair, I think it comes down to personal taste.